src/libraries/Common/tests/System/Net/Http/ResponseStreamTest.cs

```diff
@@ -307,15 +310,16 @@ public async Task BrowserHttpHandler_StreamingRequest()
                         return Task.FromResult(send);
                     }
                     return Task.FromResult(0);
-                }));
+                })), "test");
+            req.Content = content;
 
             req.Content.Headers.Add("Content-MD5-Skip", "browser");
 
             using (HttpClient client = CreateHttpClientForRemoteServer(Configuration.Http.RemoteHttp2Server))
             using (HttpResponseMessage response = await client.SendAsync(req))
             {
                 Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-                Assert.Equal(size.ToString(), Assert.Single(response.Headers.GetValues("X-HttpRequest-Body-Length")));
+                Assert.Equal((size + 129).ToString(), Assert.Single(response.Headers.GetValues("X-HttpRequest-Body-Length")));
```

Comment:
Could you please explain +129 ? Maybe made it a named constant.

Reply:
This test now sends `MultipartFormDataContent` which adds some more things like boundaries, field name ("test" in this case) and multi part headers. Will add a constant for it.